lib/Repository.js

```diff
@@ -491,7 +491,7 @@ class Repository extends Requestable {
     * @param {Requestable.callback} [cb] - will receive the pull request information
     * @return {Promise} - the promise for the http request
     */
-   updatePullRequst(number, options, cb) {
+   updatePullRequest(number, options, cb) {
```

Comment:
This has been  open for a while now. I'm going to add an alias and add a warning as suggested.